docs/theory/diffusion/diffusion.rst

```diff
@@ -0,0 +1,713 @@
+    plt.ylim([-1, 3])
+
+    plt.subplot(1, 2, 2)
+    for method in ("cve unweighted", "cve ensemble", "ols ensemble"):
+        plot_accuracy(snrs, results[method], variance=variances[method], label=method);
+    plt.ylim([-1, 3])
+
+.. image:: figures/all_estimators.png
+
+We can see that the ensemble estimates fare quite well in this case, while the single estimates show a large variability.
+A lot of this variability in the individual estimates comes from short tracks.
+We can show this by including a filtering step (requiring a minimum track length).
+We can filter tracks shorter than a certain length by using :func:`~lumicks.pylake.filter_tracks`::
+
+    min_length = 10
+    methods = {
+        "cve single": lambda list_of_tracks: [t.estimate_diffusion("cve") for tracks in list_of_tracks for t in tracks],
+        "ols single": lambda list_of_tracks: [t.estimate_diffusion("ols") for tracks in list_of_tracks for t in tracks],
+        "cve (filtered)": lambda list_of_tracks: [t.estimate_diffusion("cve") for tracks in list_of_tracks for t in lk.filter_tracks(tracks, min_length)],
+        "ols (filtered)": lambda list_of_tracks: [t.estimate_diffusion("ols") for tracks in list_of_tracks for t in lk.filter_tracks(tracks, min_length)],
+        "cve ensemble": lambda list_of_tracks: [t.ensemble_diffusion("cve") for t in list_of_tracks],
+        "ols ensemble": lambda list_of_tracks: [t.ensemble_diffusion("ols") for t in list_of_tracks],
+        "cve ensemble (filtered)": lambda list_of_tracks: [lk.filter_tracks(t, min_length).ensemble_diffusion("cve") for t in list_of_tracks],
+        "ols ensemble (filtered)": lambda list_of_tracks: [lk.filter_tracks(t, min_length).ensemble_diffusion("ols") for t in list_of_tracks],
+    }
+
+    snrs = 10**np.arange(-1, 1.1, 0.125)
+    results, variances = test_estimation_methods(simulate_tracks, snrs, methods=methods)
+
+    plt.figure(figsize=(12, 4))
+    plt.subplot(1, 2, 1)
+    for method in ("cve single", "ols single", "cve (filtered)", "ols (filtered)"):
+        plot_accuracy(snrs, results[method], variance=variances[method], label=method);
+    plt.ylim([-1, 3])
+
+    plt.subplot(1, 2, 2)
+    for method in ("cve ensemble", "ols ensemble", "cve ensemble (filtered)", "ols ensemble (filtered)"):
+        plot_accuracy(snrs, results[method], variance=variances[method], label=method);
+    plt.ylim([-1, 3])
+
+.. image:: figures/filtering_short_tracks.png
+
+The good news is that the ensemble estimates are pretty robust to filtering short tracks.
+This is expected since these estimates already incorporate weighting (assigning lower weight to short tracks).
+The bad news is that the variance for low SNR is still very high, while the method with the most precision at low SNR (OLS) is very biased.
+Unfortunately, what this means is that we either need to settle for ensemble estimates, or accept high variance results.
```

Comment:
Really nice that you consider all these different scenarios and illustrate what you have to think about when doing diffusion analysis!